sal/class.json-api-site-jetpack-base.php

```diff
@@ -89,11 +89,11 @@ function after_render_options( &$options ) {
 	}
 
 	function get_jetpack_modules() {
-		if ( is_user_member_of_blog() ) {
```

Comment:
if we remove the check from here we will want to add it in `json-endpoints/class.wpcom-json-api-get-site-endpoint.php`